halotools/mock_observables/two_point_clustering/angular_tpcf.py

```diff
@@ -205,13 +206,14 @@ def area_spherical_cap(chord):
             D2R = N2*(da*rho2)
 
             # calculate the random-random pairs.
+            # cbx: This should be NR(NR-1) if we are doing an autocorr
             rhor = NR**2/global_area
             RR = (da*rhor)
 
             return D1R, D2R, RR
 
     def pair_counts(sample1, sample2, chord_bins,
-            N_thread, do_auto, do_cross, _sample1_is_sample2):
+            num_threads, do_auto, do_cross, _sample1_is_sample2):
```

Comment:
`N_thread` is not used inside this function. Instead we use `num_threads` defined in the `angular_tpcf` namespace. It doesn't make any difference to what is actually done, but this argument should either be removed (and we just use `num_threads` in the outer namespace) or we should use the value that is passed. I've done the latter.